deploy/openshift-clusters/roles/proxy-setup/defaults/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+# Default variables for proxy-setup role
+
+# Proxy configuration
+proxy_port: 8213
+proxy_user: ec2-user
+
+# Container configuration
+proxy_container_name: external-squid
+proxy_image: quay.io/openshifttest/squid-proxy:multiarch
```

Comment:
I wonder who maintains this image and whether it will (ever) be updated in case of security issues :sweat_smile: